### ISSUE TITLE
508-defect-4 [COGNITION]: CONSIDER revising content and adding aria-describedby to online form calls to action

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12723]
## [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

@jenstrickland brought this up in Slack the other day:
> I'm working on the full 508 audit for Caregivers 10-10CG MVP. I notice that on the landing page we have a pattern that we've used elsewhere, and something occurred to me as I was reading the page with VoiceOver. As the reader moves down the h1, intro text, then the "Start your application" button, a screen reader user may never know that there are instructions for the process following the button.
>
> At first I thought maybe it would make sense to not have the button above the directions, but this pattern exists all over the site — I think. Do you think it may be helpful if we added an `aria-describedby` attribute to the button to cue that there is relevant info below it?

I think it would be beneficial to review this for ways to improve understanding for users with cognitive issues and those who use assistive technology. After discussing with @emilywaggoner we settled on the following actions:

* Emily will reach out to the Analytics team to see if there are events being tracked for the top of page and bottom of page calls to action
* Add an `aria-describedby="<UNIQUE_ID>"` attribute to the call to action buttons that would trigger screen readers to read the instructional block after they read the button text.
* Review with @peggygannon to see if there are ways we could adjust the text in the highlighted content block (screen shot below) to let screen reader users know there are instructions on the page, without degrading the experience for other users.
* Run user testing with new content and markup if warranted by analytics data

## Point of Contact

**VFS Point of Contact:** _Trevor, Jennifer_

## Possible Fixes (optional)

```html
<button
  aria-describedby="description-text"
  class="adc-c-button adc-c-button-large"
  type="button"
>
  Start your application
</button>

<p id="description-text">Follow the steps below to apply for the Program of Comprehensive Assistance for Family Caregivers.</p>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-19 at 2 24 53 PM](https://user-images.githubusercontent.com/934879/90936661-455fe480-e3cb-11ea-906e-e6865603dd73.png)

**Comment by 1Copenut** [comment]
Re-assigning to @meganhkelley because this looks like a forms issue. 

/cc @emilywaggoner

**Comment by meganhkelley** [comment]
Hello! Forms library is now the purview of the Design System team — @caw310 is the Product Manager for that team, I'll assign her here.
